### ISSUE TITLE
Regarding visualisation of algorithms.

### ISSUE THREAD

**Issue by Akshat1Nar** [Vatsal32/8Queens#1]
Your software does not visulaise the backtracing algorithm for us, instead it only displays all possible configuration...Please try to add that functionality to your codebase! :100:

**Comment by Vatsal32** [comment]
So, you mean that it should show all the intermediate steps it takes for one solution??

**Comment by Vatsal32** [comment]
Try executing gui_backtrack.py now.

**Comment by Akshat1Nar** [comment]
Yes, it works but still, animation is too fast to keep track of what is happening :crying_cat_face:
